xikolo-common/ViewHelpers/Settings.swift

```diff
@@ -11,9 +11,10 @@ import UIKit
 class Settings {
 
     class func open() {
-        if let appSettings = URL(string: UIApplicationOpenSettingsURLString) {
-            UIApplication.shared.openURL(appSettings)
-        }
+        guard let appSettings = URL(string: UIApplicationOpenSettingsURLString) else { return }
+        guard UIApplication.shared.canOpenURL(appSettings) else { return }
+        UIApplication.shared.open(appSettings, options: [:], completionHandler: nil)
+        // TODO: write test for this
```

Comment:
What are you planning to test here?

Reply:
I don't like the creation of this URL with an "Apple-String". Shouldn't break, but who knows. I know I'd never manually open the settings from within our app, so I thought about a test here in order to prevent this from breaking on a future iOS version or platform.